compiler/passes/convert-uast.cpp

```diff
@@ -126,6 +126,8 @@ struct LoopAttributeInfo {
   LLVMMetadataList llvmMetadata;
   // The @assertOnGpu attribute, if one is provided by the user.
   const uast::Attribute* assertOnGpuAttr = nullptr;
+  // The @gpu.assertEligible attribute, if one is provided by the user.
+  const uast::Attribute* assertEligibleAttr = nullptr;
```

Comment:
Not a big worry, but this variable is not self-documenting at all. Eligible for what?